src/flex-box/NASA/nasa-plz.html

```diff
@@ -0,0 +1,47 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./styles-plz.css">
+    <title>NASA Landing Page</title>
+</head>
+<body>
+    <main>
+        <h1>NASA</h1>
+    </main>
+    <footer>
+        <section>
+            <article>Social Media
+                <div class="text-container">
+                    <p>Twitter @nasa</p>
+                    <p>Facebook fb.com/nasa</p>
+                    <p>Instagram - @nasa</p>
+                    <p>Youtube - youtube.com/nasa</p>
+                </div>
+            </article>
+            <article>Office Info
+                <div class="text-container">
+                    <p>Public Communications Office</p>
+                    <p>NASA Headquarters</p>
+                    <p>300 E. Street SW, Suite 5R30</p>
+                    <p>Washington, DC 20546</p>
+                    <p>(202) 867-5309</p>
+                    <p>(202) 555-5555</p>
+                </div>
+            </article>
+            <article>Contact us
+                <div class="text-container">
+                    <p class="box-top">Name</p>
+                    <div class="space1"></div>
+                    <p class="box-top">Message</p>
+                    <div class="space2"></div>
```

Comment:
use `<textarea cols="30" rows="10"></textarea>`
mess with the cols and rows numbers for size